tests/test_views.py

```diff
@@ -301,7 +301,7 @@ def test_create_thread_message(authenticated_client):
     )
     assert (
         ai_message.message["data"]["content"]
-        == "The current temperature in San Francisco, CA is 32 degrees Celsius."
+        == "The current temperature in San Francisco, CA is 32°C."
```

Comment:
If this changed, perhaps you need to also update the vcr recordings when langgraph is false?

Reply:
Ah, I think it won't be consistent on two runs... so the test can only run with true or false. Another reason to remove the old code.

Reply:
Yep, that's it, the vcr had to be updated so there were small changes to outputs